gateway-service/src/main/java/org/zowe/apiml/gateway/service/scheme/HttpBasicPassticket.java

```diff
@@ -28,6 +30,11 @@ public AuthenticationScheme getAuthenticationScheme() {
 
     @Override
     public void apply(ServiceInstance serviceInstance, RouteDefinition routeDefinition, Authentication auth) {
+        if (StringUtils.isEmpty(auth.getApplid())) {
+            log.debug("Service {} does not have configured APPLID. The authorization scheme will be ignored", serviceInstance.getServiceId());
+            return;
+        }
```

Comment:
Should this be a warn rather?

Reply:
it will spam the log in case some service is missing this parameter, I think it's more for debug

Reply:
I guess it should be warn until we add onboarding validation - it should return 400 when APPLID is missing for passtickets...